sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsHostMemoryStore.scala

```diff
@@ -36,7 +38,10 @@ class RapidsHostMemoryStore(
   // Returns an allocated host buffer and whether the allocation is from the internal pool
   private def allocateHostBuffer(size: Long): (HostMemoryBuffer, Boolean) = {
     // spill to keep within the targeted size
-    synchronousSpill(math.max(maxSize - size, 0))
+    val amountSpilled = synchronousSpill(math.max(maxSize - size, 0))
+    if (amountSpilled != 0) {
```

Comment:
should we have a similar log line here like so:

```
        logInfo(s"Spilled $amountSpilled bytes from the host store")
```

Reply:
Updated to add a corresponding log.  Downgraded another type of logging message that is related to debug.